src/app/(auth)/(navigationsBarLayout)/mypage/ui/ModifyForm.tsx

```
@@ -0,0 +1,155 @@
+              }
+              className="w-full px-4 py-3 bg-gray-800 border border-gray-600 rounded-lg text-white focus:outline-none focus:border-green-500 transition-colors"
+              placeholder="블로그 url을 입력해주세요."
+            />
+          </div>
+          {/* 자기소개 */}
+          <div className="md:col-span-2">
+            <label className="block text-sm font-medium text-gray-300 mb-2">자기소개</label>
+            <textarea
+              value={editForm.introduction || ''}
+              onChange={(e) => setEditForm((prev) => ({ ...prev, introduction: e.target.value }))}
+              rows={4}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)